bluebutton-data-model-rif/src/main/java/gov/hhs/cms/bluebutton/data/model/rif/RifFilesEvent.java

```diff
@@ -1,31 +1,33 @@
 package gov.hhs.cms.bluebutton.data.model.rif;
 
 import java.time.Instant;
-import java.time.format.DateTimeFormatter;
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.Comparator;
 import java.util.List;
 
+
+
 /**
  * Models a RIF file event, in which a new set of RIF files have been made
  * available for processing.
  */
 public final class RifFilesEvent {
```

Comment:
With the refactoring here that moves `DataSetManifest` into this class, I'm wondering if further cleanup isn't possible? Specifically, the `DataSetManifest.getEntries()` field seems awfully similar to `RifFilesEvent.fileEvents`...

Can `RifFilesEvent.fileEvents` be removed and instead just derived from the `DataSetManifest`? I don't know if it can or not, but seems like it'd make things a lot simpler if it could.